app/experimenter/nimbus-ui/src/hooks/useExperiment.test.tsx

```diff
@@ -40,6 +40,86 @@ describe("hooks/useExperiment", () => {
       await waitFor(() => expect(hook.notFound).toBeTruthy());
     });
 
+    describe("ready for review", () => {
+      const readyMessage = {
+        public_description: ["This field may not be null."],
+        proposed_duration: ["This field may not be null."],
+        proposed_enrollment: ["This field may not be null."],
+        firefox_min_version: ["This field may not be null."],
+        targeting_config_slug: ["This field may not be null."],
+        reference_branch: ["This field may not be null."],
+        channels: ["This list may not be empty."],
```

Comment:
BTW This is a copy of an actual `message` value from the GQL query. The array of error strings have no use in our UI, though, so they're pretty much ignored across the board in practice. Maybe they could be used in the (!) icon under a tooltip or something, but that's for down the road.